docs/tutorials/deploy-vm/README.md

```diff
@@ -1,3 +1,248 @@
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+          <ComputerName>my-vm</ComputerName>
+        </component>
+      </settings>
+      <settings pass="oobeSystem">
+        <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64"
+          publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS"
+          xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+          <UserAccounts>
+            <AdministratorPassword>
+              <Value>FakePassword</Value>
+              <PlainText>true</PlainText>
+            </AdministratorPassword>
+          </UserAccounts>
+          <OOBE>
+            <HideEULAPage>true</HideEULAPage>
+            <HideLocalAccountScreen>true</HideLocalAccountScreen>
+            <HideOEMRegistrationScreen>true</HideOEMRegistrationScreen>
+            <HideOnlineAccountScreens>true</HideOnlineAccountScreens>
+            <HideWirelessSetupInOOBE>true</HideWirelessSetupInOOBE>
+            <ProtectYourPC>3</ProtectYourPC>
+            <SkipMachineOOBE>true</SkipMachineOOBE>
+            <SkipUserOOBE>true</SkipUserOOBE>
+          </OOBE>
+          <TimeZone>Central Standard Time</TimeZone>
+        </component>
+      </settings>
+      <cpi:offlineImage cpi:source="" xmlns:cpi="urn:schemas-microsoft-com:cpi" />
+    </unattend>
+```
+
+##### Minimal Config
+
+The following `Secret` resource may be used to bootstrap a Windows image with minimal information. Please note the image would have to be using a Volume License SKU as the product ID is not provided:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-vm-bootstrap-data
+  namespace: my-ns
+stringData:
+  unattend: |
+    <?xml version="1.0" encoding="utf-8"?>
+    <unattend xmlns="urn:schemas-microsoft-com:unattend">
+      <settings pass="oobeSystem">
+        <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+          <OOBE>
+            <SkipMachineOOBE>true</SkipMachineOOBE>
+            <SkipUserOOBE>true</SkipUserOOBE>
+          </OOBE>
+        </component>
+      </settings>
+    </unattend>
+```
+
+For more information on Sysprep, please refer to Microsoft's [official documentation](https://learn.microsoft.com/en-us/windows-hardware/customize/desktop/unattend/).
+
+#### vAppConfig
+
+The vAppConfig bootstrap method is useful for legacy, VM images that rely on bespoke, boot-time processes that leverage vAppConfig properties for customizing a guest.
+Refer to [Deploy a VM with vAppConfig](https://github.com/vmware-tanzu/vm-operator/blob/main/docs/tutorials/deploy-vm/vappconfig.md) for instructions to use this bootstrap method.
```

Comment:
can you also change this link to `https://vm-operator.readthedocs.io/en/latest/tutorials/deploy-vm/vappconfig/`?